src/app/dance-moves/dance-move-edit/dance-move-edit.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Params } from '@angular/router';
+
+@Component({
+  selector: 'app-dance-move-edit',
+  templateUrl: './dance-move-edit.component.html',
+  styleUrls: ['./dance-move-edit.component.scss']
```

Comment:
if you don't need dance-move-edit.component.scss file, you can delete it and also delete this line :) but this is not necessary

Reply:
Deleted :).